src/Slider.jsx

```diff
@@ -652,14 +652,14 @@ class Rheostat extends React.Component {
             />
           );
         })}
-        {PitComponent && pitPoints.map((n) => {
+        {PitComponent && pitPoints.map((n, index) => {
           const pos = algorithm.getPosition(n, min, max);
           const pitStyle = orientation === 'vertical'
             ? { top: `${pos}%`, position: 'absolute' }
             : { left: `${pos}%`, position: 'absolute' };
 
           return (
-            <PitComponent key={n} style={pitStyle}>{n}</PitComponent>
+            <PitComponent key={`${n}_PC_${index}`} style={pitStyle}>{n}</PitComponent>
```

Comment:
Why are the pit points not unique?

Regardless, if there's nothing unique about the item, just `key={index}` is sufficient - anything else is just a waste.

Reply:
oh nevermind, I just had to use a bound component function, here is notes in case somebody else gets here:
...
pitComponent={this.getPitBar}
...
and that one has access to state, i.e.:
getPitBar = ({ style, children }) => {
    let pointWithHeight = this.state.pitPointsWithHeights.find(
      x => x.key === children
    );
let height = pointWithHeight && pointWithHeight.doc_count * 10;
    console.log('height is', height);
    return (
      <div
        style={{
          ...style,
          background: 'red',
          width: 5,
          height: height,
          top: 20
        }}
      />
    );
  };